governator-jetty/src/main/java/com/netflix/governator/guice/jetty/DefaultJettyConfig.java

```diff
@@ -39,6 +45,21 @@ public String getWebAppContextPath() {
         return webAppContextPath;
     }
 
+    @Override
+    public boolean isPlaintextSocketEnabled() {
+        return plaintextSocketEnabled;
+    }
+
+    @Override
+    public int getRequestHeaderSize() {
+        return requestHeaderSize;
```

Comment:
c/b 'getRequestHeaderSizeBytes' if not a stylistic clash with other property names

Reply:
:+1: